tests/notes/test_notes_routes.py

```
@@ -353,27 +337,27 @@ async def test_contributor_can_add_but_not_edit(two_user_clients):
 
     entry = (await alice.post(f"/api/notes/{doc_id}/entries", json={"text": "alice entry"})).json()
     eid = entry["id"]
-    assert (await bob.patch(f"/api/notes/{doc_id}/entries/{eid}", json={"done": True})).status_code == 403
+    assert (await bob.patch(f"/api/notes/{doc_id}/entries/{eid}", json={"done": True})).status_code == 405
     assert (await bob.delete(f"/api/notes/{doc_id}/entries/{eid}")).status_code == 403
 
 
 @pytest.mark.asyncio
 async def test_owner_always_has_full_access(two_user_clients):
     """The owner can add, edit, and delete regardless of any member permission rules."""
     alice, bob, _ = two_user_clients
```

Ruff (0.16.1)

[warning] Unpacked variable `bob` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)